src/schemas/listing-core.json

```diff
@@ -0,0 +1,243 @@
+{
+  "$id": "http://schema.originprotocol.com/listing-core-v1.0.0",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Origin Protocol core listing schema",
+  "definitions": {
+    "moneyAmount": {
+      "type": "object",
+      "title": "An amount of money in a specified currency",
+      "properties": {
+        "currency": {
+          "type": "string"
+        },
+        "amount": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "currency",
+        "amount"
+      ]
+    },
+    "location": {
+      "type": "object",
+      "properties": {
+        "street": {
+          "type": "string"
+        },
+        "extendedStreet": {
+          "type": "string"
+        },
+        "locality": {
+          "type": "string"
+        },
+        "region": {
+          "type": "string"
+        },
+        "box": {
+          "type": "string"
+        },
+        "postalCode": {
+          "type": "string"
+        },
+        "country": {
+          "type": "string"
+        },
+        "coord": {
+          "type": "object",
+          "properties": {
+            "latitude": {
+              "type": "integer",
+              "minimum": -90,
+              "maximum": 90
+            },
+            "longitude": {
+              "type": "integer",
+              "minimum": -180,
+              "maximum": 180
+            }
+          },
+          "required": [
+            "latitude",
+            "longitude"
+          ]
+        }
+      },
+      "required": [
+        "street",
+        "locality",
+        "region",
+        "country"
+      ]
+    },
+    "media": {
+      "type": "object",
+      "title": "Representation of a media asset such as an image or a video",
+      "properties": {
+        "url": {
+          "type": "string",
+          "format": "uri"
+        },
+        "contentType": {
+          "type": "string"
+        },
+        "description": {
+          "type": "string"
+        },
+        "dimension": {
+          "type": "object",
+          "properties": {
+            "height": {
+              "type": "integer"
+            },
+            "width": {
+              "type": "integer"
+            }
+          },
+          "required": [
+            "height",
+            "width"
+          ]
+        }
+      }
+    },
+    "timeSlot": {
```

Comment:
@tyleryasaka @jordajm - In our fractional implementation, I recall that the availability data was stored in a _different_ IPFS blob from the main listing, given its editability. Is that right? 

@franckc , if that's right, I don't think it makes sense to put fractional data in listing-core.